### ISSUE TITLE
[FEATURE] show caption of a message even if the media is deleted

### ISSUE THREAD

**Issue by code-consensus** [KnugiHK/WhatsApp-Chat-Exporter#111]
- WhatsApp version: any
- OS: Android
- Platform: any
- Branch and version: 0.10.0

**Describe the bug**
Messages that have media _and_ a caption, but where the **media** has been deleted, are not shown at _all_ in the output file. The only thing shown is "The media is missing"

**Solution:**
Add a section to detect a message caption to the whatsapp.html template, specifically right after [line 139](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/b34045a59fcd4da40c1021def34415039b789f8b/Whatsapp_Chat_Exporter/whatsapp.html#L139) and [line 223](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/b34045a59fcd4da40c1021def34415039b789f8b/Whatsapp_Chat_Exporter/whatsapp.html#L223), within the section with the `{% if msg.meta == true or msg.media == false and msg.data is none %}`

Specifically something like: 
`{% if msg.caption is not none %}`
`    <div class="INSERT w3 formatting">`
`             {{ msg.caption | urlize(none, true, '_blank') }}`
`    </div>`
`{% endif %}`

**Comment by KnugiHK** [comment]
Implemented in 87b1fcc.

I noticed there are class names in your example:

> ```
> {% if msg.caption is not none %}
>     <div class="INSERT w3 formatting">
>              {{ msg.caption | urlize(none, true, '_blank') }}
>     </div>
> {% endif %}
> ```

Are there any specific reasons or things your want to achieve?

**Comment by code-consensus** [comment]
> I noticed there are class names in your example:

I put that simply as a placeholder (`class="INSERT w3 formatting"`), because I don't really know CSS! I just tried w3-container, and seems to work fine!

**Comment by code-consensus** [comment]
@KnugiHK I actually noticed your change may _also_ benefit the other area where the message caption is used in the HTML template. In [line 169](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/87b1fcc038ac26ad1e6f4f52087d0f24ff03fe65/Whatsapp_Chat_Exporter/whatsapp.html#L169) and [line 258](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/87b1fcc038ac26ad1e6f4f52087d0f24ff03fe65/Whatsapp_Chat_Exporter/whatsapp.html#L258):
```
{% if msg.caption is not none %}
<br>
{{ msg.caption | urlize(none, true, '_blank') }}
{% endif %}
```
Instead of `<br>`, wrapping that in the `<div class="w3-container">` works a lot better actually.

With the `<br>`, it looked like this for me, for example:
![msg_caption_nodiv_file](https://github.com/user-attachments/assets/19405640-b1de-4848-b40a-e9530ee1c24d)

With your `<div class="w3-container">`:
![msg_caption_div_file](https://github.com/user-attachments/assets/3364676e-6b7b-4a15-8ddc-50382b92c3ac)

**Comment by KnugiHK** [comment]
> I actually noticed your change may _also_ benefit the other area where the message caption is used in the HTML template. In [line 169](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/87b1fcc038ac26ad1e6f4f52087d0f24ff03fe65/Whatsapp_Chat_Exporter/whatsapp.html#L169) and [line 258](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/blob/87b1fcc038ac26ad1e6f4f52087d0f24ff03fe65/Whatsapp_Chat_Exporter/whatsapp.html#L258):
> 
> ```
> {% if msg.caption is not none %}
> <br>
> {{ msg.caption | urlize(none, true, '_blank') }}
> {% endif %}
> ```
> 
> Instead of `<br>`, wrapping that in the `<div class="w3-container">` works a lot better actually.
> 

I changed it but I haven't have time to test it, although I believe there should be no error. Could you please do so? Thanks!

**Comment by code-consensus** [comment]
> I changed it but I haven't have time to test it, although I believe there should be no error. Could you please do so? Thanks!

I tested these changes, and it seems to work for me just fine!

**Comment by KnugiHK** [comment]
> > I changed it but I haven't have time to test it, although I believe there should be no error. Could you please do so? Thanks!
> 
> I tested these changes, and it seems to work for me just fine!

Thanks! I will leave this open until next release.

**Comment by KnugiHK** [comment]
Released in 0.10.5.
